io_scene_niftools/modules/nif_import/armature/__init__.py

```diff
@@ -235,15 +235,20 @@ def guess_orientation(self, n_armature):
         # return string identifiers
         return ids[forward_ind], ids[up_ind]
 
+    @staticmethod
+    def argmax(values):
```

Comment:
What's the purpose of implementing this instead of np.argmax? Avoiding the import?

Reply:
Yeah that was the idea. It's probably not worth the overhead. But pulling it out makes it more readable.